fe1-web/src/core/components/TextBlock.tsx

```diff
@@ -37,13 +39,15 @@ const propTypes = {
   bold: PropTypes.bool,
   visibility: PropTypes.bool,
   color: PropTypes.string,
+  size: PropTypes.number,
```

Comment:
Hmm yes this is not optimal indeed.. Yes I agree, I think just using the `Text` component with the correct styles is a better solution!
Also with this component it is easy to create text components in a dozen different sizes / colors whereas usually in UIs you try to limit the number of different font sizes / colors etc (which each can be its own style in a StyleSheet).

Reply:
I agree that the `TextBlock` component is not practical to use at all. I struggled a little bit with it last semester and that's why I used a lot of `Text`s in the end. However, I think that having a generic text for the application is a good idea for consistency (like font and color), but it can be done with styles.

Reply:
It seems there's a consensus, let's try to slowly phase out the `TextBlock` then 😄